### PR TITLE
task 6 completed

### PR DESCRIPTION
**Pull request by maksymprokopenko**
- [x] File serverless.yml contains configuration for catalogBatchProcess function
- [x] File serverless.yml contains policies to allow lambda catalogBatchProcess function to interact with SNS and SQS
- [x] File serverless.yml contains configuration for SQS catalogItemsQueue
- [x] File serverless.yml contains configuration for SNS Topic createProductTopic and email subscription

Additional tasks:
- [ ] catalogBatchProcess lambda is covered by unit tests
- [ ] set a Filter Policy for SNS createProductTopic in serverless.yml (Create an additional email subscription and distribute messages to different emails depending on the filter for any product attribute)

FE app link: **https://d3kf9393vfqbzn.cloudfront.net/admin/products**

### CODE REVIEW

File: import-service/serverless.yml [hunk 1]
```diff
@@ -12,6 +12,11 @@ provider:
                 -   "s3:*"
             Resource:
                 -   "arn:aws:s3:::uploaded-mp/*"
+        -   Effect: "Allow"
+            Action: 
+                -   "sqs:*"
+            Resource:
+                -   "arn:aws:sqs:eu-west-1:869906325472:mp-aws-sqs-queue"
```
Comment (dartwlad) [inline import-service/serverless.yml:19]: Please export SQSQueue from products service and import here

**Comment by dartwlad** [comment]
Please provide a link to your FE app

**Comment by maksymprokopenko** [comment]
> Please provide a link to your FE app

Updated, please check the PR description.

File: import-service/serverless.yml (hunk 1 above)
Comment (maksymprokopenko) [reply import-service/serverless.yml:19]: Is this a good way to import some data from another service? Looks strange if we will start to import data higher then root holder of each service.

**Pull request merged**